index.js

```diff
@@ -578,7 +569,7 @@ PersonaClient.prototype.updateProfile = function(guid, profile, token, callback)
     try {
         _.map([guid, token], function (arg) {
             if (!_.isString(arg)) {
-               throw "guid and token are required strings";
+                throw "guid and token are required strings";
```

Comment:
It is weird that we are not using `map` as it mapping method. It should be changed to a `[guid, token].forEach(function(thing) { throw "guid and token are required strings"; });`